### ISSUE TITLE
Treat read only packages as read only

### ISSUE THREAD

**Issue by citizenmatt** [JetBrains/resharper-unity#1587]
All packages in the Read Only node of the Unity Explorer should be treated as read only, and should not allow adding new files, drag/drop, etc. The package node itself should also not be a drag source.

**Comment by van800** [comment]
Cutting read-only package and pasting to Packages folder is a valid case though.

**Comment by citizenmatt** [comment]
Yes, although that would require extra handling. We shouldn't "cut" but copy, and we'd need to make sure that all file permissions were reset to read/write.

And we'd have to take care of transitive dependencies. If read only package A depends on package B, we couldn't move package B to the `Packages` folder, because then package A wouldn't compile (unless we were generating projects for all packages, and even then Unity's own compilation might not support this scenario).
